src/Microsoft.AspNet.Routing/RouteValueDictionary.cs

```diff
@@ -31,28 +31,32 @@ public RouteValueDictionary()
         }
 
         /// <summary>
-        /// Creates a RouteValueDictionary initialized with the provided input values.
+        /// Creates a <see cref="RouteValueDictionary"/> initialized with the provided input value.
```

Comment:
"input value" -> `<paramref name="obj"/>` (and I think we use "specified" more than "provided")